website/client-old/js/components/groupTasks/groupTasksController.js

```diff
@@ -0,0 +1,147 @@
+habitrpg.controller('GroupTasksCtrl', ['$scope', 'Shared', 'Tasks', 'User', function ($scope, Shared, Tasks, User) {
```

Comment:
Yes, I was going to abstract everything to the Task Service which would be shared between the challenge, user and group. But, since we are refactoring, I tried not to change much.